test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/AllowListVerificationProviderTests.cs

```diff
@@ -1015,24 +1012,69 @@ public AllowListVerificationProviderTests(SigningTestFixture fixture)
                         signedPackageVerifier,
                         signedPackageVerifierSettings);
 
-                    // Act
-                    Action action = async () => await PackageExtractor.ExtractPackageAsync(
```

Comment:
An exception was thrown with the require mode policy, but the test was not failing because the exception was not observed.  `action` was not awaited.